auth_totp/models/res_users.py

```diff
@@ -13,6 +13,12 @@
 class ResUsers(models.Model):
     _inherit = 'res.users'
 
+    @classmethod
+    def _build_model(cls, pool, cr):
+        ModelCls = super(ResUsers, cls)._build_model(pool, cr)
+        ModelCls.SELF_WRITEABLE_FIELDS += ['mfa_enabled', 'authenticator_ids']
+        return ModelCls
+
```

Comment:
I feel that we can avoid the `_build_model` internal method, and achieve the same effect extending the `write` method instead. What do you think?